modules/hal_nordic/Kconfig

```diff
@@ -10,12 +10,19 @@ config HAS_NORDIC_DRIVERS
 menu "Nordic drivers"
 	depends on HAS_NORDIC_DRIVERS
 
+config NRF_802154
+	bool
+	help
+	  This helper symbol indicates that nRF 802.15.4 radio driver API is implemented
+	  and available in some way.
```

Comment:
The only reason I added this kconfig was to avoid doing the `OR` you suggested. I thought that it is more preferable to provide helper symbols rather than do some if/else logic in Cmake. I can change that back.